app/Factory/CompanyFactory.php

```diff
@@ -49,6 +49,11 @@ public function create(int $account_id) :Company
         $company->markdown_enabled = false;
         $company->tax_data = new TaxModel();
         $company->first_month_of_year = 1;
+
+        // default mailbox
+        $company->expense_mailbox = config('ninja.inbound_expense.webhook.mailbox_template') != '' ?
+            str_replace('{{company_key}}', $company->company_key, config('ninja.inbound_expense.webhook.mailbox_template')) : null;
+
         return $company;
```

Comment:
I don't think we need to create a mailbox name by default, this would be configured for use when a user wants to implement it. I think this solves a lot of issues like this one.

Reply:
The default mailbox name was to ensure, that the mailbox-name for expenses is generated with the company_key and the free/pro clients dont change them to anything they like.

Reply:
I think if a user has access to this, then they will have the option to configure this. I don't think it should be included in the companyfactory.